gripper_position:
  ros__parameters:
    type: effort_controllers/JointPositionController
    joint: gripper
    pid:
      p: 100.0
      i: 0.0
      d: 0.0

gripper_sub_position:
  ros__parameters:
    type: effort_controllers/JointPositionController
    joint: gripper_sub
    pid:
      p: 100.0
      i: 0.0
      d: 0.0